File: cloud/oauth-server/src/main/java/pers/acp/admin/oauth/bus/event/RefreshApplicationEvent.java
package pers.acp.admin.oauth.bus.event;

import org.springframework.cloud.bus.event.RemoteApplicationEvent;

/**
 * @author zhang by 19/03/2019
 * @since JDK 11
 */
public class RefreshApplicationEvent extends RemoteApplicationEvent {

    private static final long serialVersionUID = -4068861028403590633L;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    private String message;

    public RefreshApplicationEvent() {
    }

    public RefreshApplicationEvent(Object source, String origin, String destination, String message) {
        super(source, origin, destination);
        this.message = message;
    }

}

File: cloud/oauth-server/src/main/kotlin/pers/acp/admin/oauth/bus/event/RefreshApplicationEvent.kt
package pers.acp.admin.oauth.bus.event

import org.springframework.cloud.bus.event.RemoteApplicationEvent

/**
 * @author zhang by 19/03/2019
 * @since JDK 11
 */
class RefreshApplicationEvent(origin: String?, destination: String?, val message: String?, source: Any = Object()) : RemoteApplicationEvent(source, origin, destination)
